_sass/_components/variables.scss

```diff
@@ -29,4 +30,4 @@ $text-max-width:     60.6rem;
 
 // Miscellaneous
 $border-accent:      1rem solid $color-bright;
-$border-light:       1px solid $color-gray-lightest;
\ No newline at end of file
+$border-light:       1px solid $color-gray-lightest;
```

Comment:
Alternatively, we could move `$border-light` to `_components/layouts.scss`, the components currently using it are all after "L" in the alphabet, so it would still work since they're after layouts in the import order. However, we'd have an issue if we ever wanted to use it in something before "L." So maybe putting the variable here is a little redundant, but the safest.